butterknife-annotations/src/main/java/butterknife/Unbinder.java

```diff
@@ -0,0 +1,19 @@
+package butterknife;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+/**
+ * Bind a target field to an un-binder implementation. Recommended to be used when requesting method
```

Comment:
"Recommended to be used when" --> "Use when"

"method bindings in a" --> "binding in a"

"{@link android.app.Fragment}" --> "{@linkplain android.app.Fragment fragment}"